chapter7.html

```diff
@@ -0,0 +1,210 @@
+<html>
+  <head>
+    <title>Forms</title>
+  </head>
+  <body>
+    <form action="http://www.example.com/review.php" method="get">
+      <fieldset>
+        <legend>
+          Your Details:
+        </legend>
+        <label>
+          Name:
+          <input type="text" name="name" size="30" maxlength="100">
+        </label>
+        <br />
+        <label>
+          Email:
+          <input type="email" name="email" size="30" maxlength="100">
+        </label>
+        <br />
+      </fieldset>
+      <br />
+      <fieldset>
+        <legend>
+          Your Review:
+        </legend>
+        <p>
+          <label for="hear-about">
+            How did you hear about us?
+          </label>
+          <select name="referrer" id="hear-about">
+            <option value="google">Google</option>
+            <option value="friend">Friend</option>
+            <option value="advert">Advert</option>
+            <option value="other">Other</option>
+          </select>
+        </p>
+        <p>Would you visit us again?
+        <br />
+        <label>
+          <input type="radio" name="rating" value="yes" />
+          Yes
+        </label>
+        <label>
+          <input type="radio" name="rating" value="no" />
+          No
+        </label>
+        <label>
+          <input type="radio" name="rating" value="maybe" />
+          Maybe
+        </label>
+      </p>
+      <p>
+        <label for="comments">
+          Comments:
+        </label>
+        <br />
+        <textarea rows="4" cols="40" id="comments">
+        </textarea>
+      </p>
+      <label>
+        <input type="checkbox" name="subscribe" checked="checked" />
+        Sign me up for email updates
+      </label>
+      <br />
+      <input type="submit" value="Submit review" />
+      </fieldset>
+    </form>
+  </body>
+</html>
+
+  <form action="http://wwww.example.com/subscribe.php" method="get">
```

Comment:
lol, TOMMY! EVERYTHING HAS TO GO IN AN HTML AND THEN BODY TAG!

If you need to create separate files like chapter7a and chapter7b or something